draft-ietf-quic-transport.md

```diff
@@ -4439,11 +4441,11 @@ original_connection_id (0x0000):
   Retry packet (see {{packet-retry}}).  A server MUST include the
   original_connection_id transport parameter if it sent a Retry packet.
 
-idle_timeout (0x0001):
+max_idle_timeout (0x0001):
 
-: The idle timeout is a value in milliseconds that is encoded as an integer; see
-  ({{idle-timeout}}).  If this parameter is absent or zero then the idle
-  timeout is disabled.
+: The max idle timeout is a value in milliseconds that is encoded as an integer;
+  see ({{idle-timeout}}).  Idle timeout is disabled when both endpoints omit this
+  transport parameteter or specify a value of 0.
```

Comment:
In its current wording, this would mean that if one of the endpoints omits the parameter or sets it to 0, the resulting idle timeout would be 0 (since we're taking the minimum of both values).

Reply:
I think that we should not interpret 0→∞.  I would prefer that we omit a value when there is none and assign the value an infinite value (or 2^62-1 if you prefer) as a default.

Reply:
SG, I'm going to no longer make the default 0.